seqr/views/apis/anvil_workspace_api.py

```diff
@@ -236,7 +248,95 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
         project_name=project.name,
         sample_type=sample_type,
         genome_version=GENOME_VERSION_LOOKUP.get(project.genome_version),
+        dag_name = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=sample_type, version=DAG_VERSION),
         dag=json.dumps(pipeline_dag, indent=4),
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+def _trigger_anvil_dags(project, data_path, user_input):
+    genome_test_type = 'AnVIL_{sample_type}'.format(sample_type=user_input['sampleType'])
+
+    anvil_variables = _get_dag_variables(genome_test_type)
+    updated_anvil_variables = {
+        "active_projects": [project.guid],
+        "es_enable_export": 'true',
+        "vcf_path": data_path,
+        "project_path": '{}v1'.format(_get_loading_project_path(project, user_input['sampleType'])),
+        "projects_to_run": [project.guid],
+    }
+    projects = anvil_variables["active_projects"]
+    updated_projects = updated_anvil_variables["active_projects"]
+ 
+    _update_variables(genome_test_type, updated_anvil_variables)
+
+    dag_id = "seqr_vcf_to_es_{anvil_type}_v{version}".format(anvil_type=genome_test_type, version=DAG_VERSION)    
+    _wait_for_dag_variable_update(projects, updated_projects, dag_id)
+    
+    _trigger_dag(dag_id)
+    
+def _wait_for_dag_variable_update(projects, updated_projects, dag_id):
+    if set(projects) != set(updated_projects):
```

Comment:
as discussed in our one-on-one, we want to update this behavior to check that at least one task_id contains the project guid in it, instead of waiting for the list of tasks to change